app/boards/shields/snap/snap_right.overlay

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2022 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#include "snap.dtsi"
+
+&default_transform {
+	col-offset = <8>;
+};
+
+&kscan_composite {
+		direct {
+			kscan = <&kscan_direct>;
```

Comment:
If the direct kscan driver instance isn't used at all on the left side, just define it in this file. No need for it to clutter the shared .dtsi file.

Reply:
Thanks for the tip. What's the syntax for defining the direct instance in the right .overlay file? I'm getting a `devicetree error: /root/firmware/zmk/app/boards/shields/snap/snap_right.overlay:13 (column 15): parse error: expected label reference (&foo)` and compilation errors when defining it directly inside the kscan_composite node.

Reply:
You'd add it nested under the root node like normal, e.g.

```
/ {
   kscan_direct: kscan_direct {
     Blah blah
   };
};

&kscan_composite {
  Reference the new node here
};
```

Reply:
Aha! So simple. I didn't realize I could add the root node outside of the base dtsi. Thanks!